app/services/billable_metrics/breakdown/unique_count_service.rb

```diff
@@ -0,0 +1,121 @@
+            date: from_date_in_customer_timezone,
+            action: 'add',
+            amount: persisted_count,
+            duration: (to_date_in_customer_timezone + 1.day - from_date_in_customer_timezone).to_i,
+            total_duration: period_duration,
+          ),
+        ]
+      end
+
+      def added_breakdown
+        date_field = Utils::TimezoneService.date_in_customer_timezone_sql(customer, 'quantified_events.added_at')
+
+        added_list = prorated_added_query.group(Arel.sql("DATE(#{date_field})"))
+                                         .order(Arel.sql("DATE(#{date_field}) ASC"))
+                                         .pluck(Arel.sql(
+                                           [
+                                             "DATE(#{date_field}) as date",
+                                             'COUNT(quantified_events.id) as metric_count',
+                                          ].join(', '),
+                                         ))
+
+        added_list.map do |aggregation|
+          OpenStruct.new(
+            date: aggregation.first.to_date,
+            action: 'add',
+            amount: aggregation.last,
+            duration: (to_date_in_customer_timezone + 1.day - aggregation.first).to_i,
+            total_duration: period_duration,
+          )
+        end
+      end
+
+      def removed_breadown
+        date_field = Utils::TimezoneService.date_in_customer_timezone_sql(customer, 'quantified_events.removed_at')
+
+        removed_list = prorated_removed_query.group(Arel.sql("DATE(#{date_field})"))
+                                             .order(Arel.sql("DATE(#{date_field}) ASC"))
+                                             .pluck(Arel.sql(
+                                              [
+                                                "DATE(#{date_field}) as date",
+                                                'COUNT(quantified_events.id) as metric_count',
+                                              ].join(', '),
+                                              ))
+
+        removed_list.map do |aggregation|
+          OpenStruct.new(
+            date: aggregation.first.to_date,
+            action: 'remove',
+            amount: aggregation.last,
+            duration: (aggregation.first + 1.day - from_date_in_customer_timezone).to_i,
+            total_duration: period_duration,
+          )
+        end
+      end
+
+      def added_and_removed_breakdown
+        added_field = Utils::TimezoneService.date_in_customer_timezone_sql(customer, 'quantified_events.added_at')
+        removed_field = Utils::TimezoneService.date_in_customer_timezone_sql(customer, 'quantified_events.removed_at')
+
+        added_and_removed_list = prorated_added_and_removed_query.group(
+          Arel.sql("DATE(#{added_field}), DATE(#{removed_field})"),
+          ).order(
+          Arel.sql("DATE(#{added_field}) ASC, DATE(#{removed_field}) ASC"),
+          ).pluck(Arel.sql(
+          [
```

Comment:
Layout/FirstArgumentIndentation: Indent the first argument one step more than the start of the previous line.